api/v1/editTaskTemplates/formRequests/UpdateTaskTemplate.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace PKP\API\v1\editTaskTemplates\formRequests;
+
+use APP\core\Application;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+use PKP\editorialTask\enums\EditorialTaskDueInterval;
+use PKP\editorialTask\enums\EditorialTaskType;
+
+class UpdateTaskTemplate extends FormRequest
+{
+    public function authorize(): bool
```

Comment:
This won't work as we haven't yet integrated/adopted Laravel auth policies (it uses Laravel gates and policies according to the documentation: https://laravel.com/docs/12.x/validation#authorizing-form-requests)